### PR TITLE
List possible values when inviting a team within an environment

### PR DESCRIPTION
**Pull request by LEUNGUU**
### Feature or Bugfix

- Feature

### Detail
- Provide a drop down list of Cognito Groups for user to choose when  inviting a team to an organization or an environment

### Relates
- #78 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by dlpzx** [comment]
As an additional step, we could filter the groups in the dropdown to list all cognito groups, minus the groups that have already been invited

File: frontend/src/views/Environments/EnvironmentTeamInviteForm.js [hunk 1]
```diff
@@ -44,17 +45,15 @@ const EnvironmentTeamInviteForm = (props) => {
...
     try {
       setLoadingGroups(true);
-      const response = await client.query(
-        listEnvironmentNotInvitedGroups({
-          environmentUri: environment.environmentUri
-        })
-      );
+      console.log("fetchgroups")
+      const response = await client.query(listCognitoGroups());
+      console.log(response)
```
Comment (louishourcade) [inline frontend/src/views/Environments/EnvironmentTeamInviteForm.js:50]: remove console log

File: frontend/src/views/Environments/EnvironmentTeamInviteForm.js [hunk 2]
```diff
@@ -44,17 +45,15 @@ const EnvironmentTeamInviteForm = (props) => {
   const fetchGroups = useCallback(async () => {
     try {
       setLoadingGroups(true);
-      const response = await client.query(
-        listEnvironmentNotInvitedGroups({
-          environmentUri: environment.environmentUri
-        })
-      );
+      console.log("fetchgroups")
```
Comment (louishourcade) [inline frontend/src/views/Environments/EnvironmentTeamInviteForm.js:48]: remove console log

**Review by louishourcade — COMMENTED** [review]
Thanks for your PR @LEUNGUU, 
Can you remove the console logs in EnvironmentTeamInviteForm.js ?
Also as @dlpzx mentioned, I think it would be nice to show only groups that do have access to he environment yet. Could you add this to the PR ?

**Comment by LEUNGUU** [comment]
> Thanks for your PR @LEUNGUU, Can you remove the console logs in EnvironmentTeamInviteForm.js ? Also as @dlpzx mentioned, I think it would be nice to show only groups that do have access to he environment yet. Could you add this to the PR ?

 Hi Louis. Sure. I will look into it.

**Comment by dlpzx** [comment]
Hi @LEUNGUU, I have added the integration test for the api. For the integration test it was easier to move the cognito calls to the handlers (for mocking). I also like having all AWS boto3 calls in the handlers, even if they are calling services in the central infra account

**Comment by LEUNGUU** [comment]
@dlpzx But how do you manage to use the lambda role in this API call? If you use aws/handlers, it will try to use PivotRole. Do we create a PivotRole here in central infra account?

**Comment by LEUNGUU** [comment]
@dlpzx Even though this 'list_cognito_groups' function is not so consistent with others in handlers folder, well, I agree with you. Let's put it there.

**Review by dlpzx — APPROVED** [review]
I tested the coverage integration tests and they succeded

**Pull request merged**